### ISSUE TITLE
Ngronk Error 802

### ISSUE THREAD

**Issue by Ibims007** [inconshreveable/ngrok#799]
Today again the internal server error 802 on my device.
Anyone else?

**Comment by MAhmad88** [comment]
Today I downlod ngrok first time and put my code and it goes really smooth but then i put ./ngrok http 80 and it's not connecting it show ERR_NGROK_802 internal error. I have put --region=eu as well to make it work but not working. what to do now.

**Comment by Ibims007** [comment]
@MAhmad88 Today it is my fist day of ngronk too and I have the exact same problem.

**Comment by Diegolfa** [comment]
servers are down i think

**Comment by caroheymes** [comment]
Hello I face the same problem. here is my code
```
!pip install pyngrok --quiet
from pyngrok import ngrok

# Terminate open tunnels if exist
ngrok.kill()

# Setting the authtoken (optional)
# Get your authtoken from https://dashboard.ngrok.com/auth
NGROK_AUTH_TOKEN = mytoken
ngrok.set_auth_token(NGROK_AUTH_TOKEN )

# Open an HTTPs tunnel on port 5000 for http://localhost:5000
public_url = ngrok.connect(port="5000", proto="http", options={"bind_tls": True})
print("Tracking URL:", public_url)
```

Failed to complete tunnel connection
The connection to http://75d7-34-86-230-158.ngrok.io was successfully tunneled to your ngrok client, but the client failed to establish a connection to the local address localhost:80.

Make sure that a web service is running on localhost:80 and that it is a valid address.

The error encountered was: dial tcp 127.0.0.1:80: connect: connection refused

**Comment by Ibims007** [comment]
I think it is now working again.

**Comment by Ibims007** [comment]
Yeah the internal server error is now gone but the tunnel connetion is not working.

**Comment by Ibims007** [comment]
Now it works again! Yipi!

**Comment by Diegolfa** [comment]
> Now it works again! Yipi!

Now for some reason has a lot of ms, more than a few hours ago, more than even 2 days ago

**Comment by injarapuveerabrahmendra** [comment]
Is it working now ?

**Comment by Ibims007** [comment]
> 
> 
> > Now it works again! Yipi!
> 
> Now for some reason has a lot of ms, more than a few hours ago, more than even 2 days ago

Sorry what do you mean? What are 'ms'?

**Comment by Ibims007** [comment]
> 
> 
> Is it working now ?

Yeah it is working now.

**Comment by caroheymes** [comment]
it works perfectly ! yeah thanks !

**Comment by jarekkoziol** [comment]
For me, it doesn't work (internal server error 802) when I specify -host-header option like:
`ngrok http -host-header=rewrite 53663`
If I don't specify host-header it starts correctly

**Comment by nikitaAndriukhin** [comment]
@jarekkoziol the same issue

**Comment by conorg763** [comment]
@jarekkoziol getting same issue when using ngrok http -host-header=rewrite 8000

**Comment by RajatKul** [comment]
I am also getting same error ngrok http -host-header=rewrite 4200

**Comment by kangzeroo** [comment]
same here! `ngrok http -host-header=rewrite 4200`
using region does not fix it either:
```
ngrok http --region eu --host-header=rewrite 4200
```

**Comment by jekru** [comment]
See #802 

1. Go to https://ngrok.com/
2. Login with your account
3. Click on "Getting Started" -> "Your Authtoken"
4. Copy your authtoken
5. Execute this in your terminal: `ngrok authtoken [YOUR_AUTH_TOKEN]`

**Comment by wangruihuano** [comment]
> Hello I face the same problem. here is my code
> 
> ```
> !pip install pyngrok --quiet
> from pyngrok import ngrok
> 
> # Terminate open tunnels if exist
> ngrok.kill()
> 
> # Setting the authtoken (optional)
> # Get your authtoken from https://dashboard.ngrok.com/auth
> NGROK_AUTH_TOKEN = mytoken
> ngrok.set_auth_token(NGROK_AUTH_TOKEN )
> 
> # Open an HTTPs tunnel on port 5000 for http://localhost:5000
> public_url = ngrok.connect(port="5000", proto="http", options={"bind_tls": True})
> print("Tracking URL:", public_url)
> ```
> 
> Failed to complete tunnel connection The connection to http://75d7-34-86-230-158.ngrok.io was successfully tunneled to your ngrok client, but the client failed to establish a connection to the local address localhost:80.
> 
> Make sure that a web service is running on localhost:80 and that it is a valid address.
> 
> The error encountered was: dial tcp 127.0.0.1:80: connect: connection refused

dddd

**Comment by wangruihuano** [comment]
> 

dddddddd

**Comment by SandezMr** [comment]
./ngrok http file:///var/www/html/

**Comment by caseysoftware** [comment]
These issues are unmonitored.

For status updates, visit https://status.ngrok.com/

For community/discussion, visit: http://ngrok.com/slack

**Comment by TiagoNovelli** [comment]
ngrok creates a tunnel that puts localhost on the external network, sometimes when we are doing tests we stop the application and forget to start it on the local port, so when ngrok tries to expose our localhost it does not find anything and returns an error.
